R/conditions.R

```diff
@@ -535,6 +559,33 @@ dropTrivialFrames <- function(callnames) {
   )
 }
 
+dropTrivialTestFrames <- function(callnames) {
```

Comment:
I don't feel strongly about this, but it would make sense to me if this was just a part of `dropTrivialFrames()`.

I'm also fine keeping it separate, but if we do, let's move the call to `dropTrivialTestFrames()` to be immediately after `dropTrivialFrames(callNames)`?

Reply:
Oh, I also see now that the filtering logic needs to be a bit different.

Reply:
It started out that way, I added a big comment explaining why it needs to be the way it is.